src/Requests/GetPublicListing.php

```diff
@@ -22,12 +22,16 @@ public function resolveEndpoint(): string
     {
         $endpoint = trim(config('flatfox.endpoint', 'https://flatfox.ch'), '/');
 
-        return "$endpoint/api/v1/public-listing/?organization={$this->identifier}{$this->expand}";
+        $this->expand = array_merge($this->expand, ['documents', 'images']);
+
+        $expand = implode(',', $this->expand);
```

Comment:
@RhysLees 

`    $expand = implode(',', array_merge($this->expand, ['documents', 'images']));`